egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -69,6 +69,8 @@ def GetArgs():
     parser.add_argument('--source-sampling-rate', type=int, default=None,
                         help="Sampling rate of the source data. If a positive integer is specified with this option, "
                         "the RIRs/noises will be resampled to the rate of the source data.")
+    parser.add_argument("--include-original-data", type=str, help="If true, the output data includes one copy of the original data",
+                         choices=['true', 'false'], default = False)
```

Comment:
This doesn't look right.  If you have string-valued choices then you have to have `default = 'true'`, and check it with `args.include_original_data == 'true'`.